src/db_adapters/accounts.rs

```diff
@@ -326,3 +330,48 @@ pub(crate) async fn store_accounts_from_genesis(near_config: near_indexer::NearC
         "Accounts from genesis were added/updated successful."
     );
 }
+
+pub(crate) async fn get_lockup_ids_at_block_height(
+    pool: &actix_diesel::Database<PgConnection>,
+    block_height: u64,
+) -> Result<Vec<String>, String> {
```

Comment:
I suggest returning `Vec<AccountId>`, so the code beyond this place can safely assume that the account_id is valid.